terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1546,7 +1559,98 @@ def qq(*words):
                                 '[Install]',
                                 'WantedBy=timers.target'
                             )
-                        }
+                        },
+                        {
+                            # AWS recommends placing the amazon-cloudwatch-agent config file in this path.
+                            # The amazon-cloudwatch-agent package is also installed in this directory.
+                            'path': '/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json',
+                            'permissions': '0664',
+                            'owner': 'root',
+                            'content': json.dumps({
+                                'agent': {
+                                    # This is the frequency interval, in seconds, of the agent's file collection.
```

Comment:
Same here. And *frequency* and *interval* are inverse concepts (frequency = 1s / interval) so it doesn't make a lot of sense to use them together in this way.